src/CurrencyHelper.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace PodPoint\I18n;
+
+use Illuminate\Config\Repository;
+use NumberFormatter;
+
+class CurrencyHelper extends Helper
+{
+    /**
+     * Instance of the CountryHelper.
+     *
+     * @var CountryHelper
+     */
+    protected $countryHelper;
+
+    /**
+     * @param Repository $config
+     */
+    public function __construct(Repository $config)
+    {
+        parent::__construct($config);
+
+        $this->countryHelper = new CountryHelper($this->config);
+    }
+
+    /**
+     * Return a value in the given currency formatted for the given locale.
+     *
+     * @param float|int $value
+     * @param string $currencyCode
+     * @param string $locale
+     *
+     * @return string
+     */
+    public function toFormat($value, string $currencyCode = CurrencyCode::POUND_STERLING, string $locale = 'en')
```

Comment:
if able to use constants could use the one for the country code here too

Reply:
Yeah I've checked yesterday and it seems that we don't have constant for that. But will double check.

Reply:
ah we don't you're right... country code is different - ignore me then